packages/server/src/application/migrations/migrations/1693062232601-tags.ts

```diff
@@ -0,0 +1,31 @@
+import { getDb } from '../client';
+
+const defaultTags = [
+  'Opening',
+  'Middlegame',
+  'Endgame',
+  'Attack',
+  'Defense',
+  'Open position',
+  'Closed position',
+  'Positional play',
+  'Tactic',
+  'Mate',
+].map(text => ({ text }));
+
+export const up = async function (next: () => {}) {
+  const db = await getDb();
+  const result = await db
+    .collection('tags')
+    .findOne({ text: defaultTags[0].text });
+  if (!result) {
+    await db.collection('tags').insertMany(defaultTags);
+  }
+  next();
+};
```

Comment:
Given the example you just did, would it make sense to generalize `template.ts` and make `up()` just execute migration code after the condition is checked, migration and condition being the functions that must be defined? 

Me being nitpicky - Is there a chance that an error might happen and down should get called automatically marking the migration unsuccessful or that happens automatically?

Reply:
- I'd wait with this kind of optimisation we can easily change `template.ts` later. I guess not all migrations will have a condition or may have multiple "steps"
- Error is handled automatically by the `node-migration` library